easybuild/easyconfigs/m/MDSplus/MDSplus-7.7.12-intel-2018a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'MDSplus'
+version = '7.7.12'
+
+homepage = 'http://mdsplus.org/'
+description = """MDSplus is a set of software tools for data acquisition and storage and a methodology
+ for management of complex scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+source_urls = ['https://github.com/%(name)s/%(namelower)s/archive']
+sources = ['stable_release-%s.zip' % version.replace('.', '-')]
+checksums = ['9b781339b62464419e5c8d8166f262729307d63e6aa18799dd924300a510cdab']
+
+dependencies = [
+    ('HDF5', '1.10.1'),
+    ('libxml2', '2.9.4'),
+    ('zlib', '1.2.11'),
+    ('ncurses', '6.0'),
+    ('libreadline', '7.0')
```

Comment:
No more dependency on `Python`?

Reply:
If a system version is found it will be used but it's not a required dependency and gives more flexibility with what this module can be used with. E.g Python 2/3.